.github/workflows/build_publish.yaml

```diff
@@ -5,13 +5,26 @@ on: [push]
 jobs:
   build:
     name: build and publish voltcraft
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+
```

Comment:
Since there are no tests in this package right now, and this action is meant to run the build, we really don't need to waste compute here to test all these os-es and python versions.

It's a pure python package and building it on any of the supported python versions will create a working wheel and source distribution

Reply:
Removed